assets/js/script.js

```diff
@@ -0,0 +1,62 @@
+
+const bodySelector =  document.querySelector('body');
+const gameSection = document.querySelector('#game-section')
+
+//Update time
+
+function displayTime () {
+  const today = dayjs();
+  $('#day').text(today.format('dddd, MM/D/YY, h:mm:ss a'));
+} 
+setInterval(displayTime, 1000);
+
+// // method 1
```

Comment:
Is this needed or can the comments be removed?

Reply:
Can be removed!